modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -7,7 +7,39 @@ class EvidenceWaiverController < ClaimsApi::V2::ApplicationController
         before_action :verify_access!
 
         def submit
-          render json: { status: 'OK' }
+          # claim_id param
+          lighthouse_claim = find_lighthouse_claim!(claim_id: params[:id])
+          benefit_claim_id = lighthouse_claim.present? ? lighthouse_claim.evss_id : params[:id]          
+          bgs_claim = find_bgs_claim!(claim_id: benefit_claim_id)
+
+          if lighthouse_claim.blank? && bgs_claim.blank?
+            raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Claim not found')
+          end
+          
+          render json: { status: 'OK', success: true }
+        end
+
+        private
+
+        def bgs_service
```

Comment:
Since all these controllers have this, I wonder if we should make a ticket to merge these into the `application_controller` sometime down the road?

Reply:
[eb92f10](https://github.com/department-of-veterans-affairs/vets-api/pull/10822/commits/eb92f1054725fd171b879d4554eb6ceba8e2d866)